<launch>
  <param name="use_sim_time" value="true"/>

  <group ns="detect_track2D_and_locate">
    <rosparam file="$(find detect_and_track)/config/object_detection_drone.yaml"/>
    <rosparam file="$(find detect_and_track)/config/pose_estimator_drone_sim.yaml"/>
    <rosparam file="$(find detect_and_track)/config/image_tracker_drone_sim.yaml"/>
  </group>
  <node pkg="tf" type="static_transform_publisher" name="optitrack_to_camera" args="-0.0613 0.0249 -0.01658 -0.02913035  -0.02442605 -0.42248077  0.90557429 d435i base_link 10" />

  <node name="detect_track2D_and_locate" pkg="detect_and_track" type="detect_track2D_and_locate_node" output="screen">
    <remap from="/camera/color/image_raw" to="/uav_2/camera/color/image_raw"/>
    <remap from="/camera/aligned_depth_to_color/image_raw" to="/uav_2/camera/aligned_depth_to_color/image_raw"/>
    <remap from="/camera/aligned_depth_to_color/camera_info" to="/uav_2/camera/color/camera_info"/>
  </node>
</launch>